docker-app/qfieldcloud/core/admin.py

```diff
@@ -856,6 +857,20 @@ def __init__(self, *args, **kwargs):
         if self.instance.has_attachments_files:
             self.fields["attachments_file_storage"].disabled = True
 
+    def clean(self):
```

Comment:
Very nice addition! Not sure if it covers everything, but myself I forgot about this. Good job!